packages/syft/src/syft/store/sqlite_document_store.py

```diff
@@ -138,7 +134,7 @@ def _connect(self) -> None:
 
     def create_table(self) -> None:
         try:
-            with SyftLock(self.lock_config):
+            with self.lock:
```

Comment:
if this becomes basically a passthrough, shouldnt we add a busy_timeout then? What would prevent concurrent writes from failing in this setup?

Reply:
I think busy_timeout has a default of 30000ms, so shouldn't be an issue. I'll double check.

Reply:
Added busy_timeout = 5s